src/index.ts

```diff
@@ -1049,6 +1049,11 @@ export interface Configuration {
    */
   purposes?: Purpose[]
 
+  /**
+   * subscription configuration
+   */
+  subscriptionConfiguration?: GetSubscriptionConfigurationResponse
```

Comment:
`subscriptionConfiguration` was made available in configuration for lanyard to render the UI